src/error.rs

```diff
@@ -16,17 +14,6 @@ fn convert_error<Src>(error: crate::openapi::apis::Error<Src>, map_fn: fn(Src) -
     }
 }
 
-impl From<crate::openapi::apis::Error<apps_api::GetAppError>> for Error {
-    fn from(e: crate::openapi::apis::Error<apps_api::GetAppError>) -> Self {
```

Comment:
Will `GetApp` need to be removed from the codegen output as well?
Would that be a separate PR?

Reply:
i was planning on removing the flex sdk annotation from the schema in order to remove it in a codegen pr